src/event.rs

```diff
@@ -61,11 +63,13 @@ pub enum Event<'a, T: 'static> {
         event: WindowEvent<'a>,
     },
 
-    /// Emitted when the OS sends an event to a device.
-    DeviceEvent {
-        device_id: DeviceId,
-        event: DeviceEvent,
-    },
+    /// Emitted when a mouse device has generated input.
+    MouseEvent(device::MouseId, device::MouseEvent),
```

Comment:
I'd generally prefer not to reverse course on decisions made in #804, but I suppose a change to winit's event API conventions like this would be more appropriately introduced in a separate issue/PR.